src/main.c

```diff
@@ -403,7 +403,7 @@ int main(int argc, char **argv)
 		// let the plugins know the min update_every
 		{
 			char buf[50];
-			snprintf(buf, 50, "%d", rrd_update_every);
```

Comment:
this should be 49. Of course `rrd_update_every` will never be that long, but anyway, the right number is 49.